src/PKSim.Core/Mappers/IndividualToPopulationSettingsMapper.cs

```diff
@@ -37,7 +37,7 @@ public RandomPopulationSettings MapFrom(Individual individual)
             populationSettings.AddGenderRatio(new GenderRatio {Gender = gender, Ratio = 100 / genderCount});
          }
 
-         if (population.IsAgeDependent)
+         if (individual.IsAgeDependent)
```

Comment:
This is strictly equivalent. The `IsAgeDependent` property is new in the individual so we do not need to go to the `SpeciesPopulation` which is here called population. That way the code is also symetrical with the `IsPreterm`